crates/telio-nurse/src/qos.rs

```diff
@@ -360,7 +361,7 @@ impl Analytics {
     ///
     /// * `event` - Received WG event.
     async fn handle_wg_event(&mut self, event: &AnalyticsEvent) {
-        if event.is_from_virtual_peer() {
+        if is_from_ignored_peer(event) {
             telio_log_trace!("WG event (Virtual peer, skipped): {:?}", event);
```

Comment:
Function "ignore peer"
Log message  "Virtual peer"

Reply:
Hope it's better now 😄